example/src/app/ng-reflect-as-text/ng-reflect-as-text.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-ng-reflect-as-text',
+  templateUrl: './ng-reflect-as-text.component.html',
+  styleUrls: ['./ng-reflect-as-text.component.css']
```

Comment:
If the `styleUrls` file is empty, we can just skip this part and remove the file

Reply:
Ok. I also removed empty css files from other components.